examples/step-89/step-89.cc

```diff
@@ -517,7 +521,7 @@ namespace Step89
 
           // Perform matrix free loop and choose correct boundary face loop
           // to use Nitsche-type mortaring.
-          matrix_free.loop(&AcousticOperator::cell_loop,
+          matrix_free.loop(&AcousticOperator::cell_loop, // PM: local_apply_cell
```

Comment:
With this, you mean to rename the function?

Reply:
Yes. Similar to the other tutorials.